test/defaults.test.js

```diff
@@ -11,15 +11,15 @@ describe('Defaults', function () {
     });
     r('http://www.filltext.com/?rows=1', function (err, response, body) {
       t.strictEqual(response.statusCode, 200);
-      t.strictEqual(body[0].d, 1);
+      t.isNumber(body[0].d);
```

Comment:
As mentioned in PR comment, filltext.com does not return *1* but a random integer, e.g. try running
```bash
curl http://www.filltext.com/?rows=1&d=index
```
I my machine, it returns *17120*, so the *strictEqual* tests were failing. I changed that to *isNumber* so that they pass.